src/components/top-page/top-page-content.styles.scss

```diff
@@ -0,0 +1,38 @@
+.japanese-text {
+    color: white;
+    font: yumincho;
+    font-size: 40px;
+    font-weight: bold;
+    text-shadow: 0px 0px 16px #8E8E8E29;
+    letter-spacing: 2.8px;
+    text-align: start;
+    width: 37vw;
+    height: auto;
+}
+
+.japanese-text-container {
+    padding-top: 30vh;
+    margin-left: 19vw;
+}
+
+.english-text {
+    color: white;
+    font-family: NorthAvellion;
+    font-size: 72px;
+    text-shadow: 0px 0px 16px #8E8E8E29;
+    letter-spacing: 5px;
+    text-align: start;
+    opacity: 0.8;
+    transform: matrix(0.99, -0.16, 0.16, 0.99, 0, 0);
+    width: 55vw;
+    height: auto;
+}
+
+.english-text-big {
+    font-size: 102px;
+}
+
+.english-text-container {
+    margin-top: 22vh;
+    margin-left: 38vw;
+}
```

Comment:
maybe try using percentages to see if that helps

Reply:
Updated changes in new commits